src/components/fields/image-upload/image-upload.tsx

```diff
@@ -70,6 +77,7 @@ export const ImageUploadInner = (props: IGenericFieldProps<IImageUploadSchema>)
 					addedFrom: "schema",
 					uploadProgress: 100,
 					slot: i,
+					uploadResponse,
```

Comment:
sorry i forgot why we need this

Reply:
I think it's fine to add this, this can be used if a consumer wants to prefill with uploaded images + upload response for their own processing

Reply:
we need this when coming back to provide details page from edit page, otherwise the image cannot be loaded